tools/visual-tester/src/runner.js

```diff
@@ -102,8 +140,9 @@ module.exports = class Runner {
 
     if (this.overwrite) this.reporter.info(`Overwriting base images`);
 
-    for (const test of listOfTests) {
-      await this.runTest.bind(this)(test.url, test.options);
+    for (let i=0; i < listOfTests.length; i++) {
```

Comment:
`for` and `async` calls work a bit strange .. so below is an example of how to wait for every one of them to finish in there.

```ts
await Promise.all(listOfTests.map(async(test, index) => {
  return await this.runTest.bind(this)(test.url, test.options, index);
})
```